### PR TITLE
Changing default Graphite URL to http://127.0.0.1:80

### PR DESCRIPTION
**Pull request by deniszh**

### CODE REVIEW

File: lib/carbon/conf.py [hunk 1]
```diff
@@ -76,7 +76,7 @@
   AMQP_VERBOSE=False,
   AMQP_SPEC=None,
   BIND_PATTERNS=['#'],
-  GRAPHITE_URL='http://127.0.0.1:8000',
+  GRAPHITE_URL='http://127.0.0.1:80',
```
Comment (iksaif) [inline lib/carbon/conf.py]: The default command line to start a dev server binds on 8080, wouldn't that make more sense ?

Comment (deniszh) [reply lib/carbon/conf.py]: Good question. 8080 is also good. @DanCech ?

Comment (DanCech) [reply lib/carbon/conf.py]: Not sure what the best solution is, since in most production deployments it's going to be at 80

Comment (deniszh) [reply lib/carbon/conf.py]: I would say let's aim for prod deployments here...

**Comment by iksaif** [comment]
:+1:

**Pull request merged**